### DISCUSSION TITLE
Missing Font Textures on Fabric API 0.86.0 [1.20.1]+

### DISCUSSION THREAD

**Discussion by N4GR** [FabricMC/fabric-api#3232] [Player support] [answered]
### Description

Hello,
My issue is pretty simple and seems to be an issue someone else has also stated on the Curse Forge website; when installing `fabric-api-0.86.1+1.20.1.jar` or greater for version **1.20.1** of minecraft, my font textures appear as boxes instead of actual characters, making the game unplayable.

However, using version `fabric-api-0.85.0+1.20.1.jar` of the Fabric API works fine.

If there could be any solution to this issue, maybe I need to install some fonts I haven't got on my machine; that would be a help.

_Thank you._
![image](https://github.com/FabricMC/fabric/assets/107074986/c4acc2d9-ad9d-47ca-8643-c0162c322591)

**Comment by modmuss50** [comment, marked as answer]
Do you have optifine installed? If so try removing it.

**Reply by N4GR** [reply]
Hello, this worked, thank you.
I think for now I'm going to use the older version of Fabric-API until there is some kind of fix, Optifine is an essential for me.
